src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,6 +1,36 @@
 import React from 'react';
 import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="post" data-cy="post-info">
+      <div className="post__flex">
+        <h1 className="post__title" data-cy="post-title">
+          {title}
+        </h1>
+
+        <p className="post__body" data-cy="post-body">
+          {body}
+        </p>
+
+        <UserInfo user={user} />
```

Comment:
Should we render this component if `user === null`? Let's use conditional rendring